website_sale_secondary_unit/tests/test_website_sale_secondary_unit.py

```diff
@@ -0,0 +1,40 @@
+# Copyright 2019 Tecnativa - Sergio Teruel
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo.tests.common import HttpCase
+
+
+class WebsiteSaleHttpCase(HttpCase):
+    def setUp(self):
+        super(WebsiteSaleHttpCase, self).setUp()
+        with self.cursor() as cr:
+            env = self.env(cr)
+            self.product_ipod = env.ref('product.product_product_11')
+            self.product_ipod.write({
+                'secondary_uom_ids': [
+                    (0, 0, {
+                        'name': 'box',
+                        'uom_id': self.product_ipod.uom_id.id,
+                        'factor': 10,
+                    }),
+                    (0, 0, {
+                        'name': 'box',
+                        'uom_id': self.product_ipod.uom_id.id,
+                        'factor': 20,
```

Comment:
You can load this as demo data instead if you wish. Tours are an exception to the "rule" of not using demo data in tests, because to design them you need to have clickable viewable things, and without demo data it's very hard to fix a broken tour because you cannot run it visibly in the browser.